pkg/gobuild/gobuild.go

```diff
@@ -61,9 +61,9 @@ func GoBuild(debugname string, pkgs []string, buildflags string) error {
 
 // GoBuildCombinedOutput builds non-test files in 'pkgs' with the specified 'buildflags'
 // and writes the output at 'debugname'.
-func GoBuildCombinedOutput(debugname string, pkgs []string, buildflags string) (string, []byte, error) {
+func GoBuildCombinedOutput(debugname string, pkgs []string, buildflags, buildDir string) (string, []byte, error) {
```

Comment:
Please add a reference to the new arg to the function comment. Is debugname expected to be an absolute path at this point or will it be resolved relative to buildDir?

Reply:
Obsolete